source/CardGames.py

```diff
@@ -376,7 +394,10 @@ def marshall_first_turn(self):
       # Display the two cards the fugitive has placed, face down
       self.display_board_general()
 
-      # Main guessing loop
+      self.marshall_guess()
+
+
+    def marshall_guess(self):
```

Comment:
Looks great man. Really clean and elegant code. We made need to implement a feature that will allow the marshall to guess more than one, but not all (since I think its possible in the rules), but that isn't a necessity for now. We can always create a task later. Good work!

Reply:
Thank you sir. I didn't realize that was part of the rules, but that would be good to add later.